modules/cucumber-reportportal-formatter.js

```diff
@@ -266,13 +285,13 @@ const createRPFormatterClass = (config) => {
       this.context.stepId = this.reportportal.startTestItem(
         {
           name,
-          description,
+          description: description,
```

Comment:
Could you please describe this change?

Reply:
I don't remember why I did this, but I just tested it and was able to revert it.